src/ui/public/inspector/adapters/request/request_responder.ts

```diff
@@ -43,6 +43,13 @@ export class RequestResponder {
       ...(this.request.stats || {}),
       ...stats,
     };
+    if (this.request.startTime) {
+      const startDate = new Date(this.request.startTime);
+      this.request.stats['Request timestamp'] = {
```

Comment:
Since most of the vis parts already have been internationalized, you should use internationalization here directly:

```ts
import { i18n } from '@kbn/i18n';

const timestampKey = i18n.translate('common.ui.inspector.reqTimestampKey', {
  defaultMessage: 'Request timestamp'
});
this.request.stats[timestampKey] = {
  value: startDate.toISOString(),
  description: i18n.translate('common.ui.inspector.reqTimestampDescription', {
    defaultMessage: 'Request start timestamp'
  })
}
```